sdk/storage/AzureStorageBlob/Source/DataStructures/StorageStreamDownloader.swift

```diff
@@ -335,9 +333,7 @@ public class BlobStreamDownloader {
     /// Download the contents of this file to a stream.
     /// - Parameters:
     ///   - group: An optional `DispatchGroup` to wait for the download to complete.
-    ///   - completion: A completion handler with which to process the downloaded chunk.
-    public func next(inGroup group: DispatchGroup? = nil, then completion: HTTPResultHandler<Data>) {
```

Comment:
@tjprescott Does this look like the correct fix for this? Swiftformat flagged that we weren't using the completion parameter. It seems like if we want to know about the progress / completion of a single chunk, then we should supply a callback in the `self.options.progressCallback` property, rather than passing in the callback as a parameter to the `next` method.

Reply:
It is correct that it wasn't being called. In my branch for Monday's demo, rather than passing callbacks around, I created a `BlobDownloaderDelegate` protocol for handling this. It is much cleaner that way.

Reply:
I was thinking the same thing after seeing how large the callbacks were :)